glide.yaml

```diff
@@ -55,3 +55,7 @@ import:
 - package: github.com/go-openapi/validate
 - package: github.com/go-openapi/loads
 - package: github.com/elazarl/go-bindata-assetfs
+- package: github.com/Shopify/sarama
+  version: ^1.16.0
+- package: golang.org/x/sys
```

Comment:
There was some dependency strangeness that I debugged with Davit but couldn't figure out so we had to pin this. I'm gonna land this to unblock and I'll address this separately.